registry/testnet/interchain/squid.tokenlist.json

```diff
@@ -6,5 +6,86 @@
     "minor": 1,
     "patch": 0
   },
-  "tokens": {}
+  "tokens": {
+    "0xcf24134c93dcca4c1cfa6595dc337849d9e9e5507c3cbffd937d617e015e81f9": {
+      "tokenId": "0xcf24134c93dcca4c1cfa6595dc337849d9e9e5507c3cbffd937d617e015e81f9",
+      "deployer": "0x25A967Ab86b379356AAf9Ad8264f78EAE82D5dc9",
+      "originalMinter": null,
+      "prettySymbol": "YSLC",
+      "decimals": 18,
+      "originAxelarChainId": "moonbeam",
+      "tokenType": "interchain",
+      "deploySalt": "0x7a71c517e9782dcb798591899dc5ee04a0cd836e06ecd7edc7df5ba4ad658e14",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/yslc.svg"
```

Comment:
we had a typo in our scripts (which we'll fix). in the meantime, can you change this to `https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/yslc.svg`?

Also, if you plan on hosting a logo there, can you actually upload a `yslc.svg`?

Reply:
i fixed the URL, but still need the svg. once you upload an svg, we can merge.

Reply:
Thanks for the wait, just added a logo.